alice_dashboard/dashboard.html

```diff
@@ -0,0 +1,509 @@
+                    </div>
+                    <div class="nav-label"> Dashboard</div>
+                    <div class="nav-dropdown">
+                        <span class="icon-chevron-thick-down"></span>
+                    </div>
+                </li>
+            </a>
+            <div id="nav-pressed"></div>
+
+            <a class="sidenav-link" href="">
+                <li>
+                    <div class="nav-icon">
+                        <span class="icon-request"></span>
+                    </div>
+                    <div class="nav-label">Request</div>
+                    <div class="nav-badge">1</div>
+                    <div class="nav-dropdown">
+                        <span class="icon-chevron-thick-down"></span>
+                    </div>
+                </li>
+            </a>
+            <div id="nav-pressed"></div>
+
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-approval"></span>
+                    </div>
+                    <div class="nav-label">Approval</div>
+                </li>
+            </a>
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-procure"></span>
+                    </div>
+                    <div class="nav-label">Procure</div>
+                </li>
+            </a>
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-receive"></span>
+                    </div>
+                    <div class="nav-label">Receive</div>
+                </li>
+            </a>
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-ap"></span>
+                    </div>
+                    <div class="nav-label">Accounts Payable</div>
+                </li>
+            </a>
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-manage"></span>
+                    </div>
+                    <div class="nav-label">Manage</div>
+                </li>
+            </a>
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-report"></span>
+                    </div>
+                    <div class="nav-label">Reports</div>
+                </li>
+            </a>
+            <a href="">
+                <li class="sidenav-link">
+                    <div class="nav-icon">
+                        <span class="icon-settings"></span>
+                    </div>
+                    <div class="nav-label">Settings</div>
+                </li>
+            </a>
+        </ul>
+    </div>
```

Comment:
☝️ Good job on indentation, very easy to follow where a tag opens and where it closes